### PR TITLE
test(@vtmn/svelte): add missing component tests

### PR DESCRIPTION
**Pull request by Tlahey**
- Create all unit tests for svelte components
- Change empty value to undefined value on svelte components
- Change all class vtmx to `<VtmnIcon>` component
- Update components README
- Remove default color for `<VtmnIcon>` to prevent color errors on components

### CODE REVIEW

File: packages/sources/svelte/src/components/indicators/VtmnPrice/test/VtmnPriceWithSlots.svelte [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<script>
+  import VtmnPrice from '../VtmnPrice.svelte';
+</script>
+
+<VtmnPrice {...$$restProps}>100€</VtmnPrice>
```
Comment (GaspardMathon) [inline packages/sources/svelte/src/components/indicators/VtmnPrice/test/VtmnPriceWithSlots.svelte:5]:
```suggestion
<VtmnPrice {...$$restProps}>100 €</VtmnPrice>
```

File: packages/sources/svelte/src/components/selection-controls/VtmnChip/README.md [hunk 2]
```diff
@@ -11,3 +11,38 @@ Now, you can use `VtmnChip` on your svelte file with
 ```javascript
 <script>import {VtmnChip} from '@vtmn/svelte';</script>
 ```
+
+# How to use variants 
+## Single-choice
+
+| Props      | Description              | Type    | Required | Available values   | default value |
+|------------|--------------------------|---------|----------|--------------------|---------------|
+| selected   | If the chip are selected | boolean | true     |                    |               |
```
Comment (GaspardMathon) [inline packages/sources/svelte/src/components/selection-controls/VtmnChip/README.md:20]:
```suggestion
| selected   | If the chip is selected | boolean | true     |                    |               |
```

File: packages/sources/svelte/src/components/selection-controls/VtmnChip/README.md [hunk 3]
```diff
@@ -11,3 +11,38 @@ Now, you can use `VtmnChip` on your svelte file with
...
+| disabled   | Disable the chip         | boolean |          |                    | false         |
+| size       | Size of the chip         | string  |          | 'small' \| 'medium' | medium        |
+
+## Input
+
+| Props      | Description              | Type    | Required | Available values   | default value |
+|------------|--------------------------|---------|----------|--------------------|---------------|
+| selected   | If the chip are selected | boolean | true     |                    |               |
```
Comment (GaspardMathon) [inline packages/sources/svelte/src/components/selection-controls/VtmnChip/README.md:28]:
```suggestion
| selected   | If the chip is selected | boolean | true     |                    |               |
```

File: packages/sources/svelte/src/components/selection-controls/VtmnChip/README.md [hunk 4]
```diff
@@ -11,3 +11,38 @@ Now, you can use `VtmnChip` on your svelte file with
...
+| disabled   | Disable the chip         | boolean |          |                    | false         |
+| size       | Size of the chip         | string  |          | 'small' \| 'medium' | medium        |
+| icon       | Name of the icon         | string  |          |                    |               |
+
+## Filter
+
+| Props      | Description              | Type    | Required | Available values   | default value |
+|------------|--------------------------|---------|----------|--------------------|---------------|
+| selected   | If the chip are selected | boolean | true     |                    |               |
```
Comment (GaspardMathon) [inline packages/sources/svelte/src/components/selection-controls/VtmnChip/README.md:37]:
```suggestion
| selected   | If the chip is selected | boolean | true     |                    |               |
```

File: packages/sources/svelte/src/components/structure/VtmnList/test/VtmnListWithSlots.svelte [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  import VtmnListItem from '../../VtmnListItem/VtmnListItem.svelte';
+</script>
+
+<VtmnList {...$$restProps}>
+  <VtmnListItem index="1">
+    <svelte:fragment slot="text">Item 1</svelte:fragment>
+  </VtmnListItem>
+  <VtmnListItem index="2">
+    <svelte:fragment slot="text">Item 2</svelte:fragment>
+  </VtmnListItem>
```
Comment (GaspardMathon) [inline packages/sources/svelte/src/components/structure/VtmnList/test/VtmnListWithSlots.svelte:12]: Why is there a z-index prop here ? Is it necessary ?

**Review by GaspardMathon — CHANGES_REQUESTED** [review]

File: packages/sources/svelte/src/components/structure/VtmnList/test/VtmnListWithSlots.svelte (hunk 5 above)
Comment (Tlahey) [reply packages/sources/svelte/src/components/structure/VtmnList/test/VtmnListWithSlots.svelte:12]: Here the `VtmnListiItem` needs index for tabIndex, but yes, I will fix it to tab-index

**Review by lauthieb — APPROVED** [review]
AWESOME 😍

**Pull request merged**